README.md

```diff
@@ -7,62 +7,95 @@ djangocms-cascade
 [![Gitter chat room](https://badges.gitter.im/jrief/djangocms-cascade.svg)](https://gitter.im/awesto/djangocms-cascade)
  [![Latest version on PyPI](https://img.shields.io/pypi/v/djangocms-cascade.svg)](https://pypi.python.org/pypi/djangocms-cascade)
 
-**DjangoCMS-Cascade** is the Swiss army knife for working with Django CMS plugins.
+**DjangoCMS-Cascade** is the Swiss army knife for working with Django-CMS plugins.
 
-Documentation
--------------
+
+# Why Use DjangoCMS-Cascade?
+
+**DjangoCMS-Cascade** is a collection of plugins for Django-CMS
+[placeholders](http://docs.django-cms.org/en/develop/getting_started/tutorial.html#creating-templates).
+Instead of creating one database model for each CMS plugin, Cascade shares one database model for
+all of them. This is possible by storing the payload inside a JSON field instead of declaring each
+attribute explicitly, and it prevents us to handle all kind of nasty database migration problems.
+
```

Comment:
I'd move this paragraph to "Architecture".

Under "Why use", we should not focus on implementation details, but explain the benefit for the end-user and the site admin.